Source/WindowsServicify/WindowsServicify.Domain/WindowsBackgroundService.cs

```diff
@@ -21,12 +21,13 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
         try
         {
             _processManager.Start();
-            
+
             while (!stoppingToken.IsCancellationRequested)
             {
                 if (!_processManager.IsCorrectlyRunning())
                 {
                     _logger.LogWarning("Restarting Process...");
```

Comment:
While generally speaking this is correct the code does not explain well why 2 loggers are needed. Can you use better class/variable names?